content/blog/tdd-assessment.md

```diff
@@ -26,6 +26,15 @@ As software developers searching the internet is something of a necessity and vi
 
 With that, let's jump right in! Follow the instructions below, take your time to do it well and send us your kata once you’re happy with what you’ve done.
 
+## Things to keep in mind
+
+1. The solution needs to be hosted on a public GitHub/GitLab repository.
```

Comment:
Can we use "Host your repo on a public github/gitlab..." instead of "Needs to be.."